assets/post_install_macOS.sh

```diff
@@ -45,6 +45,9 @@ ${DSTROOT}/.mne-python/bin/conda env config vars set PYTHONNOUSERSITE=1
 logger -p 'install.info' "ℹ️ Disabling mamba package manager banner"
 ${DSTROOT}/.mne-python/bin/conda env config vars set MAMBA_NO_BANNER=1
 
+logger -p 'install.info' "ℹ️ Configuring Matplotlib to use the Qt backend by default"
+sed -i '.bak' "s/##backend: Agg/backend: qtagg/" ${DSTROOT}/.mne-python/lib/python3.10/site-packages/matplotlib/mpl-data/matplotlibrc
```

Comment:
In 6 months we'll have to remember to change this. Better to add something like the following, adapted from `extract_version.sh`
```
export PYSHORT=$($DSTROOT/.mne-python/bin/python -c "import sys; print('.'.join(map(str, sys.version_info[:2])))")
```

Reply:
... then use `$PYSHORT` in this line

Reply:
Before you do, it might be worth changing it to 3.9 to verify that CIs complain. If the installer succeeds silently/with no error, then it means this script could get broken halfway through and we'd only know once we saw some step not working in practice later...

Reply:
Yep, failing!